soul-common/src/test/java/org/dromara/soul/common/utils/LogUtilsTest.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dromara.soul.common.utils;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.function.Supplier;
+
+/**
+ * Test case for {@link LogUtils}.
+ *
+ * @author HoldDie
+ */
+public class LogUtilsTest {
+
+    private Logger logger;
+
+    private Supplier<Object> supplier;
+
+    @Before
+    public void setUp() throws Exception {
+        logger = LoggerFactory.getLogger(LogUtilsTest.class);
+        supplier = () -> "Test case for LogUtils";
+    }
+
+    @Test
+    public void testGetInstance() {
+        final LogUtils result = LogUtils.getInstance();
+        Assert.assertNotNull(result);
+    }
+
+    @Test
+    public void testDebugWithFormat() {
+        LogUtils.debug(logger, "testDebug: {}", supplier);
```

Comment:
can you assert debug info Is it what you expected？

Reply:
yes